toolkit/scss/_temporary-message.scss

```diff
@@ -7,18 +7,14 @@
 // The following placeholders must be present in your main stylesheet:
 // - shared_placeholders/_temporary-messages.scss
 
-%temporary-message-section {
-  @extend %temporary-message;
-  margin-bottom: $gutter;
-}
-
+%temporary-message-section,
 .temporary-message {
-  @extend %temporary-message-section;
+  @extend %temporary-message;
```

Comment:
Do you think it's worth importing this in this file?  

As it stands, it seems like a hidden dependency on the `shared_placeholders/_temporary-messages.scss` file.